Sources/Sham/Models/StubRequest.swift

```diff
@@ -204,4 +204,13 @@ private extension URL {
         // it must have a non-empty path or a non-nil and non-empty host
         return !self.path.isEmpty || self.host?.isEmpty == false
     }
+
+    func hasQueryStringEqualTo(url otherURL: URL) -> Bool {
+        // Split the query string apart by key/value pairs, then sort and rejoin
+        let selfQuery = self.query?.split(separator: "&").sorted(by: { $0 < $1 }).joined(separator: "&")
+        let requestQuery = otherURL.query?.split(separator: "&").sorted(by: { $0 < $1 }).joined(separator: "&")
```

Comment:
Is the conversion back to a string necessary? Or could that operation be removed an just compare 2 string arrays?